_posts/2016-04-14-using-bumbler-to-reduce-runtime-dependencies.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "Using Bumbler to reduce runtime dependencies"
+date: 2016-04-14 11:53:00
+categories: ["ruby", "rails", "performance"]
+author: "mauro-oto"
+---
+
+A few weeks ago, I ran into an interesting project called
+ [Bumbler](https://github.com/nevir/Bumbler). Bumbler lets you
+ know which are your Bundler-based project's largest dependencies.
+When you find yourself staring at the screen after running
+ `bundle exec rails c`, you may want to give this tool a try.
+
+<!--more-->
+
+Using it is simple. Add `gem 'bumbler'` to your Gemfile, run `bundle`, and
+ you're good to go by running `bundle exec bumbler`. Running this will show you
+ a progress bar and detail the dependencies which took the longest to load.
+Here's an example of a project's development dependencies:
+
+```
+➜  git:(master) ✗ bundle exec bumbler
+[#################################################                             ]
+(49/65) travis-lint...
+Slow requires:
+    110.21  render_anywhere
+    147.33  nokogiri
+    173.83  haml
+    179.62  sass-rails
+    205.04  delayed_job_active_record
+    286.76  rails
+    289.36  mail
+    291.98  capistrano
+    326.05  delayed_job
+    414.27  pry
+    852.13  salesforce_bulk_api
+```
+
+As you can see, one of the gems takes almost 1 second to load on my system,
+ and removing it increases the time it takes to run `bundle exec rails c` by
```

Comment:
@mauro-oto do you mean "it decreases the time it takes to load `bundle exec rails console`" by 1 second?